admin/partials/jws-about.php

```diff
@@ -20,22 +20,22 @@
         <img src="<?php echo esc_attr( plugins_url( '../images/icon-256x256.png', __FILE__ ) ); ?>">
 
         <h2><?php printf( esc_html__( 'Just Writing Statistics V%s', 'just-writing-statistics' ), JWS_VERSION ); ?></h2>
-        <p><?php esc_html_e( 'a fork of', ''); ?> <a href="https://wpwordcount.com">WP Word Count</a></p>
-        <p><?php esc_html_e( 'by', ''); ?> <a href="https://toolstack.com">Greg Ross</a></p>
+        <p><?php printf(esc_html__('a fork of %1$sWP Word Count%2$s', 'just-writing-statistics'),'<a href="https://wpwordcount.com" target="_blank">','</a>'); ?>
+        <p><?php printf(esc_html__('by %1$sGreg Ross%2$s', 'just-writing-statistics'),'<a href="https://toolstack.com" target="_blank">','</a>'); ?></p>
```

Comment:
Same as above.

Reply:
Keeping complete sentences is a great choice, so just adding a comment to let the translators know not to translate the developer name "Greg Ross".